### ISSUE TITLE
API Reference has changed substantially

### ISSUE THREAD

**Issue by michaelblyons** [SublimeText/sublime-text-docset#15]
Check to make sure the dashing selectors are still good. https://www.sublimetext.com/docs/api_reference.html

**Comment by FichteFoll** [comment]
Yeah, this needs an update for the new docs. Fortunately it also means we can remove some of the post-download fixes like adding anchors for each function because the new docs have them already.
